### PR TITLE
Fixes #28: Add-Custom-Font

### LINKED ISSUE #28: [Feature Request]: Add custom font
### Is your feature request related to a problem? Please describe.

Out website need custom font.

### Describe the solution you'd like

Add Open-sans as default font for website https://fonts.google.com/specimen/Open+Sans

### Describe alternatives you've considered

Gorditta font is getting used in website but its a paid font so as a replacement we are using Open-sans

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by KvngJamesNG**
Update tailwind.ocnfig.js
Update index.html
Update index.css
Update App.jsx

### CODE REVIEW

**Comment by rt4914** [comment]
I will review this PR soon, meanwhile you can work on other issues in backend and frontend @KvngJamesNG

File: src/index.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+body {
+    font-family: 'Open Sans', ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
```
Comment (rt4914) [inline src/index.css:6]: This can be simplied to `font-family: 'Open Sans', Arial, sans-serif;`

File: tailwind.config.js [hunk 2]
```diff
@@ -1,6 +1,10 @@
 /** @type {import('tailwindcss').Config} */
 export const content = ["./src/**/*.{js,jsx,ts,tsx}", "./index.html"];
 export const theme = {
-  extend: {},
+  extend: {
+    fontFamily: {
+      sans: ['Open Sans', 'ui-sans-serif', 'system-ui'],
+    },
+  },
```
Comment (rt4914) [inline tailwind.config.js:8]: Remove changes from this file, we don't need it.

File: src/App.jsx [hunk 3]
```diff
@@ -1,8 +1,10 @@
 const App = () => {
   return (
+    <div className="font-sans">
```
Comment (rt4914) [inline src/App.jsx:3]: Remove this div also, because we are applying font-family at body level, all fonts will get changed automatically.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@KvngJamesNG Suggested changes.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Looks good. Can you please reply to all open comments before merge it.

File: src/index.css (hunk 1 above)
Comment (KvngJamesNG) [reply src/index.css]: done this

File: tailwind.config.js (hunk 2 above)
Comment (KvngJamesNG) [reply tailwind.config.js]: also implemented this request

File: src/App.jsx (hunk 3 above)
Comment (KvngJamesNG) [reply src/App.jsx]: I guess the this is also not needed

**Review by rt4914 — APPROVED** [review]
Good work @KvngJamesNG 
Thanks.

**Pull request merged**